### ISSUE TITLE
Main chat-input needs accessible name (label)

### ISSUE THREAD

**Issue by iamjolly** [GSA-TTS/10x-ai-sandbox#186]
# Bug Report

## Description

**Bug Summary:**
The main chat-input field is unlabeled and only contains some placeholder-like text that is low contrast and not equivalent to an accessible name for the imput.

**Expected Behavior:**
All `input` fields need accessible names to let users of assistive technology know what they are filling out into the field.

**Actual Behavior:**
The chat-input field does not have any accessible name via `label`, `aria-label`, or `aria-labelledby` elements/attributes.

**Screenshots (if applicable):**

<img width="1273" alt="screenshot showing unlebelled chat input before chat session has started" src="https://github.com/user-attachments/assets/832fc68e-df2a-455b-b1cb-9781d074388f" />

<img width="1258" alt="screenshot showing unlabelled chat input after chat has started" src="https://github.com/user-attachments/assets/0ecd5ff3-ab00-4c9c-8e65-51408eddec7a" />

**Comment by vickimcfadden** [comment]
@mark-meyer PR review
